### PR TITLE
Add Community Fund for 2022

### LINKED ISSUE #927: Add Travel Fund allocation
I was checking our budget allocation for travel for 2022, it seems it's missing from the doc,

https://github.com/openjs-foundation/cross-project-council/blob/main/project-resources/MEMBER_TRAVEL_FUND.md#2021-openjs-foundation-board-of-directors-allocation

### PR DESCRIPTION
**Pull request by WaleedAshraf**
closes #927 

Signed-off-by: Waleed Ashraf <waleedashraf@outlook.com>

### CODE REVIEW

**Review by Trott — APPROVED** [review]
LGTM once `TBD` is filled in.

**Review by mcollina — APPROVED** [review]
lgtm

File: project-resources/MEMBER_TRAVEL_FUND.md
Comment (eemeli) [inline project-resources/MEMBER_TRAVEL_FUND.md:221]: It would be nice to have an actual number here.

Comment (WaleedAshraf) [reply project-resources/MEMBER_TRAVEL_FUND.md:221]: Yes. Waiting on review from @mhdawson.

Comment (mhdawson) [inline project-resources/MEMBER_TRAVEL_FUND.md:221]:
```suggestion
for a 2022 joint Community Fund was approved in the amount of $60,000 USD.
```

Comment (mhdawson) [reply project-resources/MEMBER_TRAVEL_FUND.md:221]: Based on the Foundation budget for 2022 I've added the number.

**Pull request merged**
